dotcom-rendering/src/lib/assets.ts

```diff
@@ -22,26 +22,37 @@ try {
  * @param {'PROD' | 'CODE' | undefined} stage the environment code is executing in
  * @returns {string}
  */
-const decideAssetOrigin = (stage: string | undefined): string => {
+const decideAssetOrigin = (
+	stage: string | undefined,
+	isDev: boolean,
+): string => {
 	switch (stage?.toUpperCase()) {
 		case 'PROD':
 			return 'https://assets.guim.co.uk/';
 		case 'CODE':
 			return 'https://assets-code.guim.co.uk/';
-		default:
-			return '/';
+		default: {
+			if (isDev) {
+				// Use absolute asset paths in development mode
+				// This is so paths are correct when treated as relative to Frontend
+				return 'http://localhost:3030/';
```

Comment:
Could we add a simple unit test for this? Just conscious that it's been inadvertently broken twice. Hopefully a unit test will flag any regressions early.

Reply:
Very good idea, have added them to this PR.